exercises/practice/gigasecond/.docs/hints.md

```diff
@@ -0,0 +1,6 @@
+# Hints
+
+# General 
+* Your code should parse a datetime object, add a gigasecond's worth of time to it, and then return the result as a datetime object. 
+
+* If you're having trouble, remember to take a look at the provided test cases under the Tests tab. These will help you figure out what the expected inputs and outputs of your function(s) should be.
```

Comment:
```suggestion
- If you're having trouble, remember to take a look at the provided test cases under the Tests tab. These will help you figure out what the expected inputs and outputs of your function(s) should be.
```

Reply:
veeery small niggle here. Our internal convention is to use `-` for bullets in markdown.  Either `*` or `-` work, we just tend to use `-`.